_includes/js/custom.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
```

Comment:
Is this file automatically included on every page, or are we going to need some frontmatter and/or page layout changes? If needed, did you want to include those changes in this PR?